llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -246,6 +246,10 @@ WebAssemblyTargetLowering::WebAssemblyTargetLowering(
                    MVT::v2f64})
       setOperationAction(ISD::SPLAT_VECTOR, T, Legal);
 
+    // Set avgceilu as legal for i8x16 and i16x8
+    // and isel will convert to AVGR_U w/ tablegen
+    setOperationAction({ISD::AVGCEILU}, {MVT::v8i16, MVT::v16i8}, Legal);
```

Comment:
Nit, you don't need the {}
```suggestion
    setOperationAction(ISD::AVGCEILU, {MVT::v8i16, MVT::v16i8}, Legal);
```